plan/expr_to_pb.go

```diff
@@ -428,13 +363,37 @@ func aggFuncToPBExpr(client kv.Client, aggFunc expression.AggregationFunction) *
 }
 
 func builtinFuncToPBExpr(client kv.Client, expr *expression.ScalarFunction) *tipb.Expr {
+	switch expr.FuncName.L {
```

Comment:
I think that if we move the switch-case to line133, `scalarFuncToPBExpr` may be very long if we support more funcs in the future. @hanfei1991